client/src/partials/price_list/pricelist.html

```diff
@@ -146,18 +149,16 @@
               <hr>
               <div class="form-group">
                   <a class="btn btn-xs btn-default" id="add_item" ng-click="PriceListCtrl.addItem()">
-                    <span class="glyphicon glyphicon-plus"></span>{{'PRICE_LIST.ADD_ITEMS' | translate}}
+                    <span class="glyphicon glyphicon-plus"></span> <span translate> PRICE_LIST.ADD_ITEMS </span>
                   </a>
               </div>
               <table class="table table-condensed">
                 <thead>
-                  <tr>
-                    <th>{{ "TABLE.COLUMNS.DESCRIPTION" | translate }}</th>
-                    <th>{{ "TABLE.COLUMNS.VALUE" | translate }}</th>
-                    <th>{{ "TABLE.COLUMNS.PERCENT" | translate }}</th>
-                    <th>{{ "TABLE.COLUMNS.INVENTORY" | translate }}</th>
-                    <th colspan="6">{{ "TABLE.COLUMNS.ACTION" | translate }}</th>
-                  </tr>
+                    <th translate> TABLE.COLUMNS.DESCRIPTION </th>
+                    <th translate> TABLE.COLUMNS.VALUE</th>
+                    <th translate> TABLE.COLUMNS.PERCENT </th>
+                    <th translate> TABLE.COLUMNS.INVENTORY </th>
+                    <th colspan="6" translate> TABLE.COLUMNS.ACTION </th>
```

Comment:
This `colspan="6"` looks wrong.   Is it really necessary?  Does the module actually use it?